config/jobs/kubeflow/kubeflow-presubmits.yaml

```diff
@@ -214,25 +214,9 @@ presubmits:
         - --workflow_file
         - build_image.yaml
     run_after_success:
-    - name: presubmit-e2e-test
+    - name: e2e-test-presubmit
```

Comment:
The convention is to start the name of ever test with the repo it belongs to. Can you change the name please?

Reply:
please follow the convention, the CI jobs in particular live in a flat namespace for logs (thankfully presubmit do not)

Reply:
done